TestIs2WavAudioPlayer/WavFileHeader.cs

```diff
@@ -0,0 +1,164 @@
+﻿using System;
+using System.Text;
+
+namespace AudioPlayer
+{
+    public class WavFileHeader
+    {
+        private readonly byte[] _header;
+
+        public WavFileHeader(byte[] header)
```

Comment:
and would be very very nice to thave the /// intelisense comments everywhere as well

Reply:
added some - hope this is ok?